### PR TITLE
[Hub Generated] Review request for Microsoft.ManagedServices to add version preview/2019-12-01-preview

### PR DESCRIPTION
**Pull request by ajay-sainy**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/ajay-sainy/azure-rest-api-specs/dev-managedservices-Microsoft.ManagedServices-2019-12-01-preview...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-12-01-preview/examples/PutRegistrationDefinition.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        "registrationDefinitionName": "DefinitionName",
+        "description": "Tes1t",
+        "managedByTenantId": "83abe5cd-bcc3-441a-bd86-e6a75360cecc",
+        "authorizations": [
+          {
+            "principalId": "f98d86a2-4cc4-4e9d-ad47-b3e80a1bcdfc",
+            "roleDefinitionId": "acdd72a7-3385-48ef-bd42-f606fba81ae7",
+            "principalIdDisplayName": "principalIdDisplayName",
+            "roleAssignmentType": "Permanent",
+            "justInTimeAccessPolicy": {
```
Comment (krnese) [inline specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-12-01-preview/examples/PutRegistrationDefinition.json]: "justInTimeAccessPolicy" cannot be used in conjunction with "roleAssignmentType": "Permanent". Only when set to "Eligible".

Comment (ajay-sainy) [reply specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-12-01-preview/examples/PutRegistrationDefinition.json]: Thanks for catching that. Fixed in the new forced commit.

**Comment by xccc-msft** [comment]
@ajay-sainy please fix PrettierCheck as per the log info.

**Review by ravbhatnagar — APPROVED** [review]
looks good

**Comment by xccc-msft** [comment]
Close this now as no response. Please feel free to reopen when you plan to continue your work.

**Pull request closed without merging**